net.brinkervii.grapejuice.metainfo.xml

```diff
@@ -27,6 +27,7 @@
 		</screenshot>
 	</screenshots>
 	<releases>
+		<release version="7.20.12" date="2023-10-14"/>
```

Comment:
Maintainers: Please add a link to the release notes.

Reply:
Unfortunately this can't be properly automated because of https://github.com/flathub/flatpak-external-data-checker/issues/222